chirp/drivers/radtel_rt620.py

```diff
@@ -0,0 +1,426 @@
+         unknown3:1,
+         chedit:1,
+         voice: 1,
+         voicesw: 1;
+      u8 backlight;
+      u8 beep:1,
+         autolock:1,
+         unknown_5b5:1,
+         noaa:2,
+         tail:1,
+         dtmfst:2;
+      u8 fminterrupt:1,
+         dualstandby:1,
+         roger:1,
+         alarmmode:2,
+         alarmtone:1,
+         fmenable:1,
+         absel:1;
+      u8 tailrevert;
+      u8 taildelay;
+      u8 tone;
+      u8 backlightbr;
+      u8 squelch;
+      u8 chbsquelch;
+      u8 chbdistype;
+      u8 chadistype;
+      u8 menuquittime;
+      u8 sk1short;
+      u8 sk1long;
+      u8 sk2short;
+      u8 sk2long;
+      u8 topshort;
+      u8 toplong;
+      u8 scanmode; // Fake Scanmode to satisfy parent method
+      u8 unknown8[5];
+    };
+    struct ani {
+      u8 unknown[5];
+      u8 code[5];
+    };
+    struct pttid {
+      u8 code[5];
+    };
+    struct proglocks {
+      u8 writelock; //0x00 - off, 0xA5 - on
+      u8 readlock;
+      char writepass[8];
+      char readpass[8];
+      u8 empty[13];
+      char unknowncode[6]; //Kill code maybe
+    };
+    struct gps {
+      u8 state;
+      u8 tzadd12;
+    };
+    struct dtmfen {
+        u8 delay;
+        u8 digit;
+        u8 interval;
+    };
+    """
+    MEM_LAYOUT = """
+    struct {
+      ul16 channels;
+      ul16 ch_a;
+      ul16 ch_b;
+    } chsel;
+    #seekto 0x0030;
+    struct {
+      struct channel mem[252];
+    } mem1;
+    #seek 0x10;
+    struct {
+      struct channel mem[255];
+    } mem2;
+    #seek 0x10;
+    struct {
+      struct channel mem[255];
+    } mem3;
+    #seek 0x10;
+    struct {
+      struct channel mem[237];
+    } mem4;
+    #seekto 0x7000;
+    struct settings settings;
+    #seekto 0x0010;
+    struct {
+      struct channel a;
+      struct channel b;
+    } vfo;
+    #seekto 0x4000;
+    struct channelname names1[372];
+    #seek 0x4;
+    struct channelname names2[372];
+    #seek 0x4;
+    struct channelname names3[255];
+    #seekto 0x7100;
+    struct gps gps;
+    #seekto 0x7069;
+    struct proglocks locks;
+    #seekto 0x8000;
+    struct pttid pttid[15];
+    struct ani ani;
+    #seekto 0x8061;
+    struct dtmfen dtmf;
+    """
+    MEM_FORMAT = MEM_DEFS + MEM_LAYOUT
+    REMOVE_SETTINGS = ['settings.scanmode']
+
+    def remove_extras(self, settings):
+        for element in settings:
+            if not isinstance(element, RadioSetting):
+                settings.remove(element)
+                settings.append(self.remove_extra_items(element))
+        settings.sort()
+
+    def remove_extra_items(self, group):
+        tmp = RadioSettingGroup(group.get_name(), group.get_shortname())
+        for element in group:
+            name = element.get_name()
+            if not isinstance(element, RadioSetting):
+                tmp.append(self.remove_extra_items(element))
+            elif name not in self.REMOVE_SETTINGS:
+                tmp.append(element)
+        return tmp
```

Comment:
I meant to mention this when I commented on the missing group thing, but.. this is a lot of work to just exclude one setting. Can we just add another class flag (like `_has_gps`, etc) and skip this setting in the base class instead of having to filter this out? If so, we can do a cleanup after the fact.

Reply:
Sorry, I was away from computer over the weekend until now. Of course, as I said, it was adapted from the code for UV-110D, and at the moment when I did not want to touch the original driver. I as you have already merged this, I will look into what can be done and make a PR later during the first December week